### DISCUSSION TITLE
Visited charging stations in Visited.

### DISCUSSION THREAD

**Discussion by DrMichael** [teslamate-org/teslamate#1020] [Ideas]
**What would you like to be added**:

Visited charging stations in Visited.

**Why is this needed**:

Not strictly needed, but would be nice.  If someone could show me, how I get those points in the TrackMap panel, I would take a look.

**Additional context**:
As seen at Teslalogger... Courtesy bassmaster.
![image](https://user-images.githubusercontent.com/8087682/96335701-bc87c100-107a-11eb-9673-0cc7f4d2f10a.png)

**Comment by DrMichael** [comment]
Ah, ok, Bassmaster has forked it for TeslaLogger (https://github.com/pR0Ps/grafana-trackmap-panel/issues/42#issuecomment-712248119). It is pretty neat. Question: should we use his fork?

**Comment by tabascoeye** [comment]
Should be an option that can be enabled/disabled. But I'd like it as well :)

**Comment by Janus00** [comment]
Yes please :-)

**Comment by roadrash2108** [comment]
I'm skeptical regarding using the fork, because this increases the effort to keep up2date with TrackMap - maybe as now 2 projects require the same features a PR to enhance TrackMap is more effective?
